src/views/IndexView.js

```diff
@@ -1,17 +1,29 @@
 'use strict'
 
-import { useContext, useEffect } from 'react'
+import React from 'react'
 import { hot } from 'react-hot-loader'
+import { autorun } from 'mobx'
 
-import RootStoreContext from '../context/RootStoreContext'
+import RootContext from '../context/RootContext'
 
 function IndexView () {
-  const { uiStore } = useContext(RootStoreContext)
+  const { networkStore, uiStore, setAppState } = React.useContext(RootContext)
 
-  useEffect(() => {
+  React.useEffect(() => {
```

Comment:
Not quite sure why you didn't keep the separate imports here, but it works so I won't require any changes :)